src/ncp-spinel/SpinelNCPInstance.cpp

```diff
@@ -380,6 +381,80 @@ static int unpack_jam_detect_history_bitmap(const uint8_t *data_in, spinel_size_
 	return ret;
 }
 
+static int
+unpack_thread_off_mesh_routes(const uint8_t *data_in, spinel_size_t data_len, boost::any& value)
+{
+
+	int ret = kWPANTUNDStatus_Ok;
+	std::list<std::string> result;
+
+	while (data_len > 0)
+	{
+		spinel_ssize_t len;
+		struct in6_addr *route_prefix = NULL;
+		uint8_t prefix_len;
+		bool is_stable;
+		uint8_t flags;
+		bool is_local;
+		bool next_hop_is_this_device;
+
+
+		len = spinel_datatype_unpack(
+			data_in,
+			data_len,
+			SPINEL_DATATYPE_STRUCT_S(
+				SPINEL_DATATYPE_IPv6ADDR_S      // Route Prefix
+				SPINEL_DATATYPE_UINT8_S         // Prefix Length (in bits)
+				SPINEL_DATATYPE_BOOL_S          // isStable
+				SPINEL_DATATYPE_UINT8_S         // Flags
+				SPINEL_DATATYPE_BOOL_S          // IsLocal
+				SPINEL_DATATYPE_BOOL_S          // NextHopIsThisDevice
+			),
+			&route_prefix,
+			&prefix_len,
+			&is_stable,
+			&flags,
+			&is_local,
+			&next_hop_is_this_device
+		);
+
+		if (len <= 0) {
+			ret = kWPANTUNDStatus_Failure;
```

Comment:
We seems to use this error type from other tasks in case of a spinel parsing/unpacking error (e.g., in `SpinelNCPTaskSendCommand` for type parser which is basically used for all `get_prop`).